src/popup/utils/utils.ts

```diff
@@ -149,6 +149,9 @@ export function toURL(url: string): URL {
 }
 
 export function truncateAddress(address: string): [string, string] {
+  if (!address) {
```

Comment:
The typing for the argument does not allow the `address` to be falsy. Please update the typing by using ternary or `string | null` depending on the case.